chain/bitcoind.go

```diff
@@ -523,11 +537,15 @@ func (c *BitcoindClient) socketHandler(zmqClient *gozmq.Conn) {
 	c.onClientConnect()
 
 	// Get initial conditions.
-	bs, err := c.BlockStamp()
+	bestHash, bestHeight, err := c.GetBestBlock()
 	if err != nil {
 		log.Error(err)
 		return
 	}
+	bs := &waddrmgr.BlockStamp{
+		Height: bestHeight,
+		Hash:   *bestHash,
```

Comment:
Should we also be setting the timestamp here?

Reply:
As you said, it's set below for subsequent blocks, but I've gone ahead and added the timestamp setting here as well for future use.